compiler/front_end/module_ir_test.py

```diff
@@ -4025,29 +3956,25 @@ def _check_source_location(source_location, path, min_start, max_end):
     result = []
     start = None
     end = None
-    if not source_location.HasField("start"):
+    if not source_location.start:
         result.append("{}.start missing".format(path))
     else:
         start = source_location.start
-    if not source_location.HasField("end"):
+    if not source_location.end:
         result.append("{}.end missing".format(path))
     else:
         end = source_location.end
 
     if start and end:
```

Comment:
I'm not sure this is checking what we want now? Maybe `is not None` is more corect.